python/metatensor-torch/metatensor/torch/atomistic/openmm_interface.py

```diff
@@ -0,0 +1,235 @@
+from typing import Iterable, List, Optional
+
+import torch
+
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import (
+    ModelEvaluationOptions,
+    ModelOutput,
+    System,
+    load_atomistic_model,
+)
+
+
+try:
+    import openmm
+    import openmmtorch
+    from NNPOps.neighbors import getNeighborPairs
+
+    HAS_OPENMM = True
+except ImportError:
+
+    class MLPotential:
+        pass
+
+    class MLPotentialImpl:
+        pass
+
+    class MLPotentialImplFactory:
+        pass
+
+    HAS_OPENMM = False
+
+
+def get_metatensor_force(
+    system: openmm.System,
+    topology: openmm.app.Topology,
+    path: str,
+    extensions_directory: Optional[str] = None,
+    forceGroup: int = 0,
+    atoms: Optional[Iterable[int]] = None,
+    check_consistency: bool = False,
+) -> openmm.System:
+
+    if not HAS_OPENMM:
+        raise ImportError(
+            "Could not import openmm and/or nnpops. If you want to use metatensor with "
+            "openmm, please install openmm-torch and nnpops with conda."
+        )
+
+    model = load_atomistic_model(path, extensions_directory=extensions_directory)
+
```

Comment:
Could you print `model.metadata()` to the log somewhere? This prints information about the model. 

Also, is there a mechanism to register papers to cite in OpenMM? If so, we should register the model references.

Reply:
I think they have some MD loggers, but I don't think they're accessible from our function (they're attached to a `Simulation` object that is independent). Literally zero clue regarding the citation mechanism

Reply:
Perhaps the best solution is to `print()` and whatever